TTHAnalysis/python/plotter/susy-ewkino/2lss/includes/fakes/mca_fakes_data.txt

```diff
@@ -0,0 +1,50 @@
+fakes_appldata : DoubleEG_Run2016B_23Sep2016_v3_runs_273150_275376_part1      ; FakeRate="susy-ewkino/2lss/fakeRate-2lss-frdata.txt", Label="Fakes",  FillColor=ROOT.kAzure+1, NormSystematic=0.30
```

Comment:
I didn't notice before, but we should keep the same labeling for the processes. Please change "Fakes" -> "nonprompt e/#mu" (people don't like the word "fakes" on public plots).